addons/t9n/static/src/core/project_list.js

```diff
@@ -0,0 +1,81 @@
+import { Component, onMounted, useState } from "@odoo/owl";
+
+import { formatList } from "@web/core/l10n/utils";
+import { useService } from "@web/core/utils/hooks";
+
+export class ProjectList extends Component {
+    static props = {};
+    static template = "t9n.ProjectList";
+
+    setup() {
+        this.action = useService("action");
+        this.state = useState({
+            filters: {
+                searchText: "",
+            },
+            sorting: {
+                column: "name",
+                order: "asc",
+            },
+        });
+        this.store = useState(useService("t9n.store"));
+        onMounted(() => {
+            this.store.fetchProjects();
+        });
```

Comment:
The call to this.store.fetchProjects() could simply be done in the setup, no need to wait for the component to be mounted (i.e. no need to put it in the onMounted hook)
```suggestion
        this.store.fetchProjects();
```